src/sentry/static/sentry/app/views/organizationGroupDetails/header.tsx

```diff
@@ -247,13 +247,15 @@ class GroupHeader extends React.Component<Props, State> {
           event={event}
         />
         <NavTabs>
-          <ListLink
-            to={`${baseUrl}${location.search}`}
-            isActive={() => currentTab === TAB.DETAILS}
-            disabled={hasGroupBeenReprocessedAndHasntEvent}
-          >
-            {t('Details')}
-          </ListLink>
+          <GuideAnchor target="issue_details">
```

Comment:
<img width="679" alt="Screen Shot 2021-03-23 at 4 21 18 PM" src="https://user-images.githubusercontent.com/16394317/112231524-1d9c5d80-8bf4-11eb-9692-bd6fa33fb7ae.png">

Reply:
@maheskett hmmm what's the issue exactly?

Reply:
the placement and styling of the details tab

Reply:
@maheskett that would have been a bad bug! Fixed:

![Screen Shot 2021-03-23 at 5 19 27 PM](https://user-images.githubusercontent.com/8533851/112235611-3d378400-8bfc-11eb-9254-9fd7f54f6448.png)